plugins/shell-proxy/README.md

```diff
@@ -45,8 +59,4 @@ The working path of **Method 2** can be changed via `$CONFIG_PROXY`
 
 ## Maintainer
 
-- <https://github.com/septs>
-
-## The oh-my-zsh plugin (shell-proxy)
-
-Public Domain
```

Comment:
plz keep the license statement

Reply:
That's not a license statement. What do you mean by Public Domain?

Reply:
Hey @septs, should I just add this back? I'm asking whether there's a more clear licensing statement that you need here. What's the meaning of that to you? Is it like a `do whatever you want with this` kind of license?

Note that the project is licensed under the MIT license, which kinda already does that, just making sure that the copyright statement is kept. You can [browse these other licenses if you need](https://choosealicense.com/licenses/). Anyway, let me know.

Reply:
ACK